doc/_static/js/custom.js

```diff
@@ -26,17 +26,26 @@ $(document).ready(function () {
     // TODO: this should listen for click events in the whole terminal
     // window, not only in the content sub-window
     // Clicking on the terminnals copies the content to the clipboard
-    $("div.highlight").click(function () {
-        if ("jquery: ", $(this).parent().attr('class') == "doctest highlight-default notranslate") {
-            let text = $(this).text();
+    $("[class*='highlight-']") .click(function () {
+        var formatCopiedText = function(text) {
```

Comment:
please change to let, var usage isn't recommended after ES 5 or 6 I think

Reply:
fixed